docs/compilers/osv.md

```diff
@@ -16,3 +29,161 @@ Compiling Java on the OSv platform requires the following parameters be met:
 * Either:
   * Project packaged as a fat `.jar` file or `.war` file *or*
   * Project uses **Gradle** or **Maven** and able to be built as a fat `.jar` or `.war`
+
+## Node.js
+It's trivial to deploy your Node.js application into unikernel. First you must collect all
+external npm libraries with:
+```
+$ cd $PROJECT_ROOT
+$ npm install
+```
+Then you can consult built-in documentation to see what configuration files are needed by UniK:
+```
+$ unik describe-compiler
+   --base osv
+   --language nodejs
+   --provider openstack
+
+OVERVIEW
+Language "nodejs" allows you to run your NodeJS 4.4.5 application.
+Please provide meta/run.yaml file where you describe your project
+structure. See below for more details.
+
+HOW TO PREPARE APPLICATION
+Install all libraries using `npm install`.
+
+CONFIGURATION FILES
+------ /meta/run.yaml ------
+config_set:
+   conf1:
+      main: <relative-path-to-your-entrypoint>
+config_set_default: conf1
+
+------ /manifest.yaml ------
+image_size: "10GB"  # logical image size
+```
+
+As you can see above, you need to create folder `meta` inside your application and create
+file `run.yaml` in it. In this file you specify where your entrypoint file is. For example, if your
+entrypoint file is `server.js`, then the final content of `meta/run.yaml` would be:
+```
+config_set:
+   conf1:
+      main: /server.js
+config_set_default: conf1
+```
+Optionally, you can create file `manifest.yaml` directly inside your project root directory
+where you specify some additional parameters. Parameter `image_size` specifys filesystem size
+of the created unikenrel.
+
+### Build command
+Use this command to build unikernel with your NodeJS application:
+```
+$ cd $PROJECT_ROOT
+$ unik build
+   --name myImg
+   --path ./
+   --base osv
+   --language nodejs
+   --provider openstack
+```
+
+## Native
+Native runtime can be used either for running your C/C++ application or for running
+arbitrary precompiled application that exists in public repository (these are called
+application packages).
+
+### Running application package
+Consult built-in documentation to see what configuration files are needed by UniK:
+```
+$ unik describe-compiler
+   --base osv
+   --language native
+   --provider openstack
+
+OVERVIEW
+Language "native" allows you to either:
+a) run precompiled package from the remote repository
+b) run your own binary code
+c) combination of (a) and (b)
+In case you intend to use packages from the remote repository, please provide
+meta/run.yaml where you list desired packages.
```

Comment:
Shouldn't this be `meta/package.yaml`? Furthermore, please add a reference to the additional documentation.